doc/transform_methods/hansenlaw.rst

```diff
@@ -8,20 +8,21 @@ Hansen-Law
 Introduction
 ------------
 
-The Hansen and Law transform is the work of E. W. Hansen and P.-L. Law [1].
+The Hansen and Law transform [1, 2] is a fast (linear time) Abel transform.
```

Comment:
What does "linear time" mean? We know that they are "linear", in the sense of linear algebra, but do we know that they are linear with time? I think that we showed that the HL algorithm scales like N^2.x, so it's pretty close to linear with the total number of pixels, but probably not quite linear.